cypress/integration/BoardingPassesInfo.js

```diff
@@ -0,0 +1,67 @@
+// @noflow
+/* global
+  Cypress,
+  cy
+*/
+/* eslint-disable jest/valid-expect */
+
+describe('Boarding Passes Info', () => {
+  beforeEach(() => {
+    cy.visit('/');
+    cy.get('[data-cy=btn-existent-booking]').click();
+    cy.get('[data-cy=link-kiwi-login]').click();
+
+    cy.get('[data-cy=input-email]').type(Cypress.env('TEST_USER_EMAIL'));
+    cy.get('[data-cy=input-password]').type(Cypress.env('TEST_USER_PASSWORD'));
+    cy.get('[data-cy=btn-sign-in]').click();
+  });
+
+  it('should display boarding passes info', () => {
+    cy.get('[data-cy=btn-boarding-passes]').click();
+
+    cy
+      .get('.boardingPassesCard')
+      .find('p')
+      .contains(
+        'Here, you can download your boarding passes or see when they will become available.',
+      )
+      .get('.boardingPassesRow')
+      .should('exist');
+  });
+
+  it('should show a specific FAQ', () => {
+    cy
+      .get('[data-cy=btn-boarding-passes]')
+      .click()
+      .get('[data-cy=faq-article-link]')
+      .contains('PNR numbers');
+  });
+
+  it('should show the previous root category after clicking home', () => {
+    cy
+      .wait(2000)
+      .get('[data-cy=faq-categories]')
+      .find('a')
+      .first()
+      .then($faqCard => {
+        cy
+          .get('[data-cy=btn-boarding-passes]')
+          .click()
+          .get('.boardingPassesCard');
```

Comment:
IMHO this `get()` has no function here, right?

Reply:
that's right, I have used it just to make sure, that after the click the view has changed, but the presence of the home breadcrumb is also indicating this change, so I've removed it 🙂